core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -740,6 +740,7 @@ private[kafka] class Processor(val id: Int,
       close(channel.id)
     }
     selector.close()
+    removeMetric("IdlePercent", Map("networkProcessor" -> id.toString))
```

Comment:
Can we add a constant for the metric name?

Reply:
We don't seem to use constants for other metric names, looks odd to have just for this one?

Reply:
I don't see any other metrics in this method. Generally, if the same magic value is used in 2 places, we should definitely use a constant. We don't follow this rule consistently, which is a shame.